engine/apps/alerts/migrations/0020_auto_20230711_1532.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.2.20 on 2023-07-11 15:32
+
+from django.db import migrations, models
+import django_migration_linter as linter
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('alerts', '0019_auto_20230705_1619'),
+    ]
+
+    operations = [
+        linter.IgnoreMigration(),
+        migrations.RemoveField(
+            model_name='alertgroup',
+            name='active_cache_for_web_calculation_id',
+        ),
+        migrations.RemoveField(
+            model_name='alertgroup',
+            name='estimate_escalation_finish_time',
+        ),
+        migrations.RemoveField(
+            model_name='alertgroup',
+            name='cached_render_for_web',
+        ),
```

Comment:
The `DROP COLUMN` operation supports the "Instant" algorithm in MySQL v8.0 so this should be a very quick operation ([docs](https://dev.mysql.com/doc/refman/8.0/en/innodb-online-ddl-operations.html#:~:text=Table%C2%A015.18%C2%A0Online%20DDL%20Support%20for%20Column%20Operations))